backend/selectgen.ml

```diff
@@ -363,6 +364,7 @@ end = struct
 
   let effect (e, _ce) = e
   let coeffect (_e, ce) = ce
+  let effect_and_coeffect e ce = e, ce
```

Comment:
Isn't here that we have two projections and a builder
with the same naming?

Reply:
wow, this is incredibly confusing, thanks for pointing it out! I've renamed the offender.